pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+
+# Rationale
+<a name="rationale"></a>
+PHEP 1 is based on [PEP 1](https://peps.python.org/pep-0001/), which is a process that has credibility in the Python community and has proven appropriate for a fairly large group, the PyHC community being (by definition) larger than the community for any one PyHC package. From this basis, changes were made to make the process relevant to PyHC and to streamline where practicable.
+
+# Rejected Ideas
+<a name="rejected-ideas"></a>
+
+## Acceptance without reference implementation
+<a name="acceptance-without-reference-implementation"></a>
+In contrast to PEP 1, any reference implementation must be complete before a PHEP is brought to a vote: there is no "Accepted" state between Draft and Final. This simplifies flow and avoids defining a process for moving from Accepted to Final, particularly if the implementation results in changes.
+
+Effort may go into a reference implementation for a standard that then gets voted down, or requires major changes to be accepted. This can be avoided by getting substantial input before starting the reference implementation and allowing the standard and implementation to evolve together. Implementation is also likely to be smaller effort (relative to the standard) for PyHC than for the Python language.
+
+## ASCII encoding
+<a name="ascii-encoding"></a>
+As originally proposed PHEPs were limited to ASCII where practicable. This was deemed overly restrictive; authors should have some freedom of format and style, and it's up to the authors and editors to ensure this is both reasonable and consistent within a PHEP. CommonMark specifies Unicode without an encoding.
+
+## Mutability of PHEPs
+<a name="immutability-of-pheps"></a>
+PHEPs are [immutable](#phep-maintenance) once Final, requiring replacement rather than modification. This maintains clarity of reference, "PHEP-x" rather than "PHEP-x version y", and avoids confusion arising from rolling updates. It encourages getting the PHEP correct before first acceptance. Finally, it removes the need to define and document an update process.
+
+A PHEP can be updated by introducing a new PHEP that starts with and builds on the original text. Everything is then open for discussion. This encourages complete re-evaluation if the community reaches the point where something is important enough to revisit, and discourages fixing a typo that doesn't hurt anything.
+
+On the downside, immutability discourages very small but potentially useful changes. It could make more work by requiring submission of a new PHEP.
+
+## Quorum requirement
+<a name="quorum-requirement"></a>
+The [resolution process](#phep-review-resolution) does not require a quorum for a vote. Suggested quorum requirements included:
+
+* Requiring "substantial community representation" at a vote.
+* Requiring representation from at least one core package at a vote.
+* Requiring the author of the PHEP to at least be present for the vote.
```

Comment:
I think this was mentioned in the 11/27 telecon, but I do believe the PHEP author should be present at the vote.

Reply:
There's nothing like marking an open issue as resolved for bringing out the discussion :) It was a suggestion that I put in open issues and there was no discussion, so I took the initiative, but happy to take further input.

My feeling is that the author has gone through the work to try and make this happen, it's probably in their own best interest to be there. People can always vote no if they think the author's absence is messing things up something terrible. But do we want to make it a hard requirement right in the process? If someone gets sick or has a family emergency and can't call in, vote gets canceled for next time?

@rebeccaringuette , I think this one was your suggestion, do you have strong feelings on it as a requirement?

Reply:
I see no need to rush a process, and there is likely a great deal to be absent from the conversation if the author isn't about to make it. I would prefer it to be a requirement, but with the option that the author can designate a replacement for themselves at the meeting if desired, or ask for the vote to be postponed.

Reply:
I feel like “should” is probably sufficient.

Reply:
Wording is [here now](https://github.com/heliophysicsPy/standards/blob/9351f28718c05a00147c54ab5dedf511f240b78b/pheps/phep-0001.md?plain=1#L132)